### PR TITLE
Windows support

### PR DESCRIPTION
**Pull request by trotzig**

### CODE REVIEW

File: lib/__tests__/findMatchingFiles-test.js [hunk 1]
```diff
@@ -6,103 +6,121 @@ import rimraf from 'rimraf';
...
-    word = 'foo';
-    existingFiles = [];
-    lookupPath = tmpDir;
-
-    subject = () => {
-      existingFiles.forEach((file) => {
-        const fullPath = `${tmpDir}/${file}`;
-        mkdirp.sync(path.dirname(fullPath));
-        fs.closeSync(fs.openSync(fullPath, 'w')); // create empty file
+['node', 'find'].forEach((finder) => {
```
Comment (lencioni) [inline lib/__tests__/findMatchingFiles-test.js]: I'm not a big fan of dynamic tests, but I understand why you are doing it here.

File: lib/findMatchingFiles.js [hunk 2]
```diff
@@ -45,3 +33,35 @@ export default function findMatchingFiles(
...
+  if (lookupPath === '') {
+    // If lookupPath is an empty string, the `find` command will not work
+    // as desired so we bail early.
+    throw new Error(`lookup path cannot be empty (${lookupPath})`);
+  }
+
+  const formattedVarName = formattedToRegex(variableName);
+  const validFilesRegex = `(/|^)${formattedVarName}(/index)?(/package)?\\.js.*`;
+
+  if (/^win/.test(process.platform)) {
```
Comment (lencioni) [inline lib/findMatchingFiles.js]: If we don't see any meaningful performance differences between these methods, we might want to simplify and just use node for this.

**Comment by lencioni** [comment]
I think this fixes #99.

File: lib/findMatchingFiles.js (hunk 2 above)
Comment (trotzig) [inline lib/findMatchingFiles.js]: I'm seeing some pretty significant differences locally. I'll whip up some numbers too when I have time.

File: lib/__tests__/findMatchingFiles-test.js (hunk 1 above)
Comment (trotzig) [inline lib/__tests__/findMatchingFiles-test.js]: Yeah. I wanted to have the same test coverage though, without duplicating things. I think this is okay.

**Pull request merged**